### ISSUE TITLE
Watch crash on every first start on a given day on some watches

### ISSUE THREAD

**Issue by Artaud** [urbandroid-team/Sleep-as-Android-Garmin-Addon#17]
As mentioned by @MrFly in https://github.com/urbandroid-team/Sleep-as-Android-Garmin-Addon/issues/10#issuecomment-443465789

On some watch models the first start on a given day always results in an app crash (IQ logo shows on the watch screen). 

Further start attempts do not crash the app - the app is able to work normally.

Whoever is able to reproduce it, please comment with your watch model and ideally attach a crash log from the watch (/GARMIN/APPS/LOGS/CIQ_LOG.txt).

**Comment by Artaud** [comment]
@MrFly, does it still happen?

**Comment by MrFly** [comment]
@artaud yes, still happens, but not every time. I think it happens on 4 of 5 starts...

**Comment by CWempe** [comment]
Does is only happen when tracking is started from phone?

If so, this seems to be a duplicate of #4.
